readme.md

```diff
@@ -26,6 +32,14 @@ To install Administrator as a Composer package to be used with Laravel 4, simply
 )
 ```
 
+### Laravel 5
+
+Then publish the config file with `php artisan vendor:publish`. This will add the file `app/config/administrator.php`. This [config file](http://administrator.frozennode.com/docs/configuration) is the primary way you interact with Administrator.
+
+Next to that all the assets, views and translation files will be published too.
+
+### Laravel 4
+
```

Comment:
@javichito I'm not sure manual inclusion of those will be the way to go, but I haven't looked at it too closely yet. If possible, I would prefer either just referencing that middleware explicitly in the routes or registering the route middleware separately from the Kernel.

Could you point out where those two facades are used? I completely forget where. In any case, if they are being used, they should either be explicitly defined via full namespace (i.e. not aliases), or the facades should just not be used at all.